packages/ui-components/style/index.css

```diff
@@ -15,7 +15,7 @@ html {
 }
 
 body {
-  color: #182026;
+  color: unset;
   font-family: var(--jp-ui-font-family);
```

Comment:
We should remove this line - see comment about inherited styles in main comment thread.

Reply:
This change will likely require us to hunt through the UI and find all the case where the font family is not set properly though.

Reply:
Note that jlab already sets that body style: https://github.com/jupyterlab/jupyterlab/blob/86ac08051d573af947c4a51554a85fd5abf96246/packages/application/style/index.css#L13-L19
I think you're implying that that css should not be in jlab either.

Reply:
Yes ;-)

Reply:
If you consider this a 1.0 issue, would you mind opening a new issue for this, and tagging it UX and 1.0?